### ISSUE TITLE
Support for Delta Lake

### ISSUE THREAD

**Issue by algattik** [AbsaOSS/spline#263]
Delta Lake is included in Databricks and the recommended format for many application especially in streaming contexts. It has been recently open-sourced at https://delta.io/.

In Azure Databricks:
```
val events = spark.read.json("/databricks-datasets/structured-streaming/events/")
events.write.format("delta").mode("overwrite").save("/delta/events")
spark.sql("CREATE TABLE events USING DELTA LOCATION '/delta/events/'")
```

Causes the Spline UI to subsequently fail to load, presumably because of #246.

The lineage stored does not contain the path:
```
{
	"_id" : "ln_b79f3eb3-ef66-4a7b-a706-28a09034eb13",
	"appId" : "app-20190602125939-0000",
	"appName" : "Databricks Shell",
	"timestamp" : 1559482785798,
	"sparkVer" : "2.4.0",
	"writeIgnored" : false,
	"rootOperation" : {
		"_id" : {
			"$binary" : "O0+PENVq59hruDF9Y/ExuA==",
			"$type" : "03"
		},
		"_t" : {
			"$binary" : "BQl0b0IxCTvvptyUiERlCNaru4xtEU6B0UM=",
			"$type" : "00"
		},
		"mainProps" : {
			"_id" : {
				"$binary" : "O0+PENVq59hruDF9Y/ExuA==",
				"$type" : "03"
			},
			"name" : "SaveIntoDataSourceCommand",
			"inputs" : [ ],
			"output" : {
				"$binary" : "e0pm77M+n7cT6zSQoCgGpw==",
				"$type" : "03"
			}
		},
		"rawString" : "SaveIntoDataSourceCommand com.databricks.sql.transaction.tahoe.sources.DeltaDataSource@403aea38, Map(path -> /delta/events), Overwrite"
	},
	"rootDataset" : {
		"_id" : {
			"$binary" : "e0pm77M+n7cT6zSQoCgGpw==",
			"$type" : "03"
		},
		"schema" : {
			"attrs" : [ ]
		}
	}
}
```

Using Spline 0.3.8.

**Comment by wajda** [comment]
Seems to be a duplicate of #246 that should be fixed in `release/0.3`
Can you retest it please?

**Comment by algattik** [comment]
Still same issues with release/0.3. No 'path' attribute gets written, and REST API fails with swallowed exception.

**Comment by wajda** [comment]
Ok, so it seems to be another corner case then. Unfortunately the team is currently fully focused on 0.4 development and we have no capacity to continue supporting 0.3 especially for supporting different corner cases and technology combinations.
So I'm marking it as "help needed" letting the community to address this issue.

**Comment by wajda** [comment]
Will be fixed in #323
